android/src/com/nvidia/devtech/NvEventQueueFragment.java

```diff
@@ -77,10 +77,10 @@ public void onCreate(Bundle savedInstanceState)
   {
     super.onCreate(savedInstanceState);
     
-    final DisplayMetrics dm = getActivity().getResources().getDisplayMetrics();
-    final float exactDensityDpi = (dm.xdpi + dm.ydpi) / 2;
-    mDisplayDensity = (int)exactDensityDpi;
-    
+    final DisplayMetrics metrics = new DisplayMetrics();
+    getActivity().getWindowManager().getDefaultDisplay().getMetrics(metrics);
```

Comment:
http://stackoverflow.com/questions/6377195/proper-way-to-get-displaymetrics-getresources-or-getwindowmanager